schema_editor/app/scripts/app.js

```diff
@@ -24,6 +24,7 @@
      */
     angular.module('ase', [
```

Comment:
Ah, ok. I agree, it should only be a few characters. I don't have any other great suggestions. Perhaps 'ash', but I don't feel strongly about it.